src/schemaCompare/schemaCompareWebViewController.ts

```diff
@@ -61,7 +61,9 @@ export class SchemaCompareWebViewController extends ReactWebviewPanelController<
     constructor(
         context: vscode.ExtensionContext,
         vscodeWrapper: VscodeWrapper,
-        node: any,
+        sourceNode: any,
```

Comment:
Ideally, these would be defined types (or unions) instead of `any`

Reply:
This can happen in a later PR.

Reply:
same here